elizhahn/karaoke-stretch
# Feature/add home navlink

## What does this PR do (summary of changes)?

 - Adds a NavLink element to the Navigation component that links back to the landing page
 - Applies dynamic classes and conditional rendering to keep the link from rendering while user is on the landing page
 - Applies CSS grid to horizontally align Navigation child elements while keeping responsivity

## How should it be tested?

 - User should not see nor be able to interact with this link on pageload (landing page)
 - User should be able to observe the Navigation bar at the top of the screen having a 'Home Page' link
 - User can click the link to be taken back to the landing page

## Future Iterations:
 - SCSS architecture requires further imports of partials to component scss files
 - SongCards still need to be given drop shadow and remove the border
 - Send the navigation bar to the bottom on mobile responsivity
